packages/react/__tests__/responsive.test.tsx

```diff
@@ -31,6 +31,32 @@ describe('responsive', () => {
     expect(el.clientWidth).toBe(250);
   });
 
+  it('Should respect single step and ignore default width of 250', async function () {
+    const component = mount(
+      <ResponsiveHelper>
+        <AdvancedImage cldImg={cloudinaryImage} plugins={[responsive({steps: 251})]} />
```

Comment:
Can you please add a test of the opposite situation? (step < container)

Reply:
Added a test and improve the comments on the test.